src/createServer.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { router: usersRouter } = require('./routes/user');
+const { router: expensesRouter } = require('./routes/expenses');
+const usersService = require('./services/users');
+const expensesService = require('./services/expenses');
+
+const cors = require('cors');
+
+const express = require('express');
+
+function createServer() {
+  const app = express();
+
+  usersService.resetStore();
+  expensesService.resetStore();
```

Comment:
Why do we need it?

Reply:
actually, we don't need
it was needed in the previous task to pass tests

I'll get rid of it